tests/cupy_tests/lib_tests/test_strided_tricks.py

```diff
@@ -29,6 +29,38 @@ def test_rolling_window(self, xp):
         return a_rolling
 
 
+class TestSlidingWindowView:
+    @testing.numpy_cupy_array_equal()
+    def test_1d(self):
+        arr = testing.shaped_arange((9,))
+        window_size = 3
+        arr_view = sliding_window_view(arr, 2)
+        return arr_view
+
+    @testing.numpy_cupy_array_equal()
+    def test_2d(self):
+        arr = testing.shaped_arange((3,4))
+        window_shape = (2,2)
+        arr_view = sliding_window_view_cp(arr, window_shape=window_shape)
+        return arr_view
+
+    @testing.numpy_cupy_array_equal()
+    def test_2d_with_axis(self):
+        arr = testing.shaped_arange((3,4))
+        window_shape = 3
+        axis = 0
+        arr_view = sliding_window_view_cp(arr, window_shape,axis)
+        return arr_view
```

Comment:
```suggestion
```suggestion
    @testing.numpy_cupy_array_equal()
    def test_2d_with_axis(self, xp):
        arr = testing.shaped_arange((3,4), xp)
        window_shape = 3
        axis = 0
        arr_view = stride_tricks.sliding_window_view(arr, window_shape,axis)
        return arr_view
```